http/server.go

```diff
@@ -529,6 +530,25 @@ func (h *DrandHandler) Health(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(b)
 }
 
+func (h *DrandHandler) ChainHashes(w http.ResponseWriter, r *http.Request) {
+	chainHashes := make([]string, 0)
+	for chainHash := range h.beacons {
+		if chainHash != common.DefaultChainHash {
+			chainHashes = append(chainHashes, chainHash)
+		}
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Cache-Control", "no-cache")
+
+	resp := make(map[string][]string)
+	resp["chains"] = chainHashes
+
+	w.WriteHeader(http.StatusOK)
+	b, _ := json.Marshal(resp)
+	_, _ = w.Write(b)
```

Comment:
More of a question than a comment: what happens if the fails write ? Do we need to catch the error and set it manually to the HTTP ResponseWriter ?

Reply:
this is the pattern from the other handlers, and i think is fine to follow here.
there will already be a log if the client hangs up the request during the writing of the response at the http server level. we probably don't need another one within the handler. there's nothing actionable we'd do because of it.

Reply:
Exactly, I took the same pattern from other handlers.